### DISCUSSION TITLE
metals-emacs apparently caches information about projects I would like to forget.

### DISCUSSION THREAD

**Discussion by frgomes** [scalameta/metals#6958] [Q&A]
I'm using Emacs.
In particular, I use treemacs for project navigation.

I've noticed that, even after I remove projects from treeemacs, even after I remove by hand folders like .bloop, .bsp, .metals and target for every single project I've removed from treemacs... I've noticed that lsp-metals-doctor-run still reports the removed project and, as it would be expect, complains that it cannot find more information for these removed projects.

I'm using metals-emacs v1.4.1.

Apparently, metals-emacs is caching references to projects somewhere, but it doesn't realize that I don't need those projects anymore. Any ideas where metals-emacs find these references to projects, please?

Thanks

**Comment by tgodzik** [comment]
The projects might be cached since they were never refreshed, we query projects on connecting to a build server. You would need to reload or run the command to connect to build server and then those should disappear.
